character.py

```diff
@@ -35,12 +35,44 @@ def handler(bot, update):
     elif text.startswith('/weapons'):
         response = get_weapons(text,db)
     elif text.startswith('/talk'):
-        response = talk(bot, update)
+        response = talk(text)
+        bot.delete_message(chat_id=update.message.chat_id, message_id=update.message.message_id)
+    elif text.startswith('/s'):
```

Comment:
For the sake of consistency (and avoid ambiguity in the future), let's add the full name to the commands. /s -> /say, /y -> /yell and /w -> whisper

Reply:
I made them that way at the beginning but i thought that one letter will be faster to write. Fixed!